default.nix

```diff
@@ -1,4 +1,7 @@
-{ pkgs ? import (fetchTarball "https://github.com/NixOS/nixpkgs/archive/23.11.tar.gz") {}
+{ pkgs ? import (fetchTarball {
+		url = "https://github.com/NixOS/nixpkgs/archive/23.11.tar.gz";
+		sha256 = "1ndiv385w1qyb3b18vw13991fzb9wg4cl21wglk89grsfsnra41k";
+	}) {}
```

Comment:
Why is this marked as a draft?

Unindent here and for `nixGLChannel` and I'll merge this immediately.

Reply:
I marked it as a draft after doing some local testing and finding that this change alone does not fully satisfy purity. I'll update the PR shortly.

What kind of indentation did you have in mind? The double indent does look a bit odd, but to me, it seemed more odd to put the ending punctuation at the same level as the top-level opening brace.

Reply:
That's the convention I've been following, as you can see with `hawkSourceInfoDevBuild`. Once the official style guide is finished I'll consider adopting it.